### PR TITLE
Add figure link to CN status heatmap

### PR DESCRIPTION
**Pull request by cansavvy**
### Purpose

I'm adding a link to the CN-status heatmap final figure and some brief words describing what the figure represents and the current threshold of 75%.

I am unsure what big picture words to say about this figure at this point. 

#### Questions for reviewers? 
- Mayhaps some of the words I added should be in a caption? 
- Are there details missing? 

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### CODE REVIEW

**Comment by cansavvy** [comment]
> I think that most of the content here makes more sense in a caption than in the text of the Results section.

Will do. 

> I'm also wondering about reconsidering the 10Mb limit that we put in: That seems quite small, and it has been long enough that I don't remember what it looked like without it, or with a larger limit. (Maybe both should be shown?)I hate to go back to this figure, but this makes me think we made a mistake excluding those regions without annotating them somehow. Maybe they could be a lighter color?

Here's where we had discussed this in Slack and where older version of the plot are posted: https://alexslemonade.slack.com/archives/CNH4FND1C/p1583435262259900 
Annotating like you are suggesting would require almost an entire overhaul of this plot, to the point where I think it would be better to make the long segments into their own plot panel. 

> If we do decide to keep that limit, it should be prominently noted that the figure only displays _small_ copy number variants. That caveat should be in the text, with the detailed explanation in the figure caption.

Will add as well

> Lastly, why "~1Mb"? isn't it exactly this (except perhaps at the ends of the chromosomes?). If not, we should explain what causes the variation (excluded regions?)

It's never *exactly* 1Mb because of how the binning works with [`Genomics::TileGenome()`](https://www.rdocumentation.org/packages/GenomicRanges/versions/1.24.1/topics/tileGenome) and the ends of the ends of the chromosomes as you mentioned. 
```
The desired tile width. The effective tile width might be slightly different but is guaranteed to never be more than the desired width.
```

> Lastly lastly: you should add "interpretability" to `custom-dictionary.txt` as described here https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/.github/PULL_REQUEST_TEMPLATE.md#spellcheck-step

Will do this.

**Comment by jashapiro** [comment]
> Here's where we had discussed this in Slack and where older version of the plot are posted: https://alexslemonade.slack.com/archives/CNH4FND1C/p1583435262259900
> Annotating like you are suggesting would require almost an entire overhaul of this plot, to the point where I think it would be better to make the long segments into their own plot panel.

Thank you for pointing me to the slack conversation!

Yeah, I figured that making two kinds of plots would be a big change. I think having two panels for now is probably the right move, and perhaps upping the size limit (double it? 5x?). I'm also just curious if any changes to the way we have processed upstream data might affect the unfiltered plot. In particular, we [recode the gains and losses for ploidy here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd), and swapping in that data (I am assuming we don't currently use it) might improve this plot. (All of this is of course separate from this PR!)

**Comment by cansavvy** [comment]
@jashapiro , I've documented your suggestions from the comment above here: https://github.com/cansavvy/openpbta-notebook-concept/issues/7

Because those are analysis changes and not something I can do on this repo, they are something I'll have to deal with on the analysis repo when we return to this issue. 

Regarding the changes to this manuscript PR, do you have any other changes you think should be done before we merge?

File: content/04.results.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ Figure {@fig:mutational-landscape-overview}B-C show concordance of these samples
 
 ![Mutational Landscape](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/fig2-mutational-landscapes.png){#fig:mutational-landscape-overview}
 
+### Copy Number Variant Overview
+
+![This figure shows dominant statuses for small copy number consensus segments (< 10 Mb) across the genome, where each square represents a ~1 Mb binned section of the genome.
+A dominant status is declared if one status is >75% coverage.
+Unstable indicates multiple non-neutral statuses totaling coverage >75%.
+The large copy number segments larger than 10 Mb have been removed from the figure for easier interpretability.](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/cn_status_heatmap.png){#fig:copy-number-status-heatmap width="7in"}
```
Comment (jashapiro) [inline content/04.results.md:32]:
```suggestion
Copy number segments longer than 10 Mb have been removed from the figure for easier interpretability.](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/cn_status_heatmap.png){#fig:copy-number-status-heatmap width="7in"}
```

**Pull request merged**